pkg/tc/generator.go

```diff
@@ -98,9 +102,21 @@ func (s *SimpleTCGenerator) genDropFilters(rule policyrules.Rule) []tctypes.Filt
 	return s.genFilters(rule.IPCidrs, rule.Ports, PrioDrop, tctypes.NewGenericActionBuiler().WithDrop().Build())
 }
 
+// genDefaultFilters generates default filters as follows:
+//  1. drop ip traffic
+//  2. drop ipv6 traffic
+//  3. drop 802.1Q ipv4 traffic
+//  4. drop 802.1Q ipv6 traffic
+func (s *SimpleTCGenerator) genDefaultFilters() []tctypes.Filter {
+	return s.genFilters(nil, nil, PrioDefault,
+		tctypes.NewGenericActionBuiler().WithDrop().Build())
+}
+
 // genFilters generates (flower) Filters based on provided ipCidrs, ports on the given prio with the given action
 // the filters generated are: matching on {ipCidrs} [X {Ports}] With priority `prio`, and action `action`
-// if no IPs and Ports provided, returned filters will match all ipv4, ipv6 traffic
+// if no IPs and Ports provided, returned filters will match all ipv4, ipv6, 802.1q traffic with provided action
+//
+//nolint:funlen
 func (s *SimpleTCGenerator) genFilters(ipCidrs []*net.IPNet, ports []policyrules.Port, prio uint16,
```

Comment:
I think it is better to refactor this function. We already have https://github.com/k8snetworkplumbingwg/multi-networkpolicy-tc/issues/42, so not a blocker